reactions/src/SPT.cpp

```diff
@@ -749,6 +789,32 @@ switch (a) {
         return 0;
 }}
 
+// BILL MOD
+double SPT::PLOOPn2_nu(int a, double vars[], double k, double err) const{
+  IOW iow;
+double loop, tree;
+double prefac = k*k*k/(4*M_PI*M_PI);
+real KMAX = QMAXp/k;
+real KMIN = QMINp/k;
+double c[2] = {KMIN,-1.};
+double d[2] = {KMAX, 1.};
+switch (a) {
+  case 0:
+    iow.initn_lin(vars[0], k, vars[1],vars[2], vars[3],vars[4]);
+    tree = P_L(k);
+    return tree;
+  case 1:
+    loop = prefac*Integrate<2>(bind(ploopn2_mgdd_nu,cref(P_L),vars,k,std::placeholders::_1,std::placeholders::_2), c, d, err);
+    tree = P_L(k);
+    return loop+tree;
+  case 8:
```

Comment:
we should document the cases a bit (comment on what they represent -- also, why case 8 after case 1?)

Reply:
The cases in PLOOPn2_nu() correspond to their equivalents in PLOOPn2(). There is some documentation about what each case represents above PLOOPn2() -- cases 4/5/6/7 correspond to P(k) in IDE but have been removed so the documentation needs updating.

Reply:
Added in comments and changed case 8 to the next integer case consistently in SPT.cpp and HALO.cpp